client/stylesheets/_toCat.scss

```diff
@@ -5184,16 +5184,66 @@ On click the images will be displayed at normal size to complete the effect
 
 .headerSearchRadiusLogo {
     margin-right: 30px;
-    font-weight: 700;
+    font-weight: 400;
 }
 
 .headerSearchRadiusLogo a {
 	font-size: 1.3rem !important;
 }
 
+.headerSearchNavLogo {
+	margin-left: 16px;
+	margin-top: 3px;
+}
+
 /* End */
 
+/* Nav Search */
 
+.headerSearchNavField {
+	color: #404040;
+    font-weight: 400;
+    font-size: 0.9rem !important;
+}
+
+.headerSearchNavField i {
+	color: #404040 !important;
+	border-bottom: 2px solid #404040;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs